projects/ngrx.io/content/guide/migration/v15.md

```diff
@@ -260,6 +260,8 @@ import { getMockStore } from '@ngrx/store/testing';
 const mockStore = getMockStore();
 ```
 
+AFTER:
+
```

Comment:
good catch! 👌

It would be also good to add `(Introduced in v15.4)` to the title (line 254).

Reply:
@markostanimirovic it should be good now, thanks!